dev_notes.md

```diff
@@ -43,3 +34,15 @@ cd ..
 git add h3lib
 git commit ...
 ```
+
+## Releasing a new version
+
+- update `CHANGELOG.md` to reflect any changes since the last release
+- update the `h3-py` version in `src/h3/_version.py`
+- create, get reviews, and merge a PR with these changes
+- go to https://github.com/uber/h3-py/releases and "Draft a new release"
+    - set the tag version and the release to the version. e.g., `v3.7.2`
```

Comment:
Why not just `git tag v3.7.2 && git push origin --tags` instead?

Reply:
my own `git` ignorance? :/

I'll add this to the notes!

Reply:
Note that this won't create a "Release" in Github - that's still a separate step after creating the tag.